package.json

```diff
@@ -29,6 +29,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@octokit/rest": "^15.9.4",
```

Comment:
can we also remove the dependency on `github-api` [here](https://github.com/qooxdoo/qooxdoo-compiler/blob/ab7866fc5060f5062407975f7ef4613c13213a73/package.json#L57) also?

Reply:
Not yet - `qx contrib update` still relies on the old library, and that still works. It's on my list but as long as there are no problems, it's low priority...

Reply:
`qx contrib publish` was broken, that's why I had to switch, but it makes total sense to get rid of github-api completely. I might have a look at the GitHub's new, GraphQl-based API (v4), which might speed up `qx contrib update` considerably, as you can make compound queries instead of having to make several queries sequentially...